src/main/rx/cc2500_frsky_shared.c

```diff
@@ -319,6 +417,7 @@ rx_spi_received_e frSkySpiDataReceived(uint8_t *packet)
 
     switch (protocolState) {
     case STATE_INIT:
+        DEBUG_SET(DEBUG_RX_FRSKY_SPI, 3, protocolState);
```

Comment:
It will be way cleaner if you just put one of these statements right after the end of the `switch` statement.

Reply:
Cleaned that up.  Was having the DEBUG_RX_FRSKY_SPI, 3 do double duty elsewhere for debugging, but that isn't needed now.